src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/xunitrunner-perf.py

```diff
@@ -244,19 +288,21 @@ def _run_xunit_from_execution(settings, test_dll, xunit_test_type, args):
         args
     )
 
-    if xunit_test_type == xunit.XUNIT_CONFIG_PERF:
+    if xunit_test_type == xunit.XUNIT_CONFIG_PERF_WINDOWS or xunit_test_type == xunit.XUNIT_CONFIG_PERF_LINUX:
         # perf testing has special requirements on the test output file name.
         # make a copy of it in the expected location so we can report the result.
         perf_log = os.path.join(test_location, "latest-perf-build.xml")
         log.info("Copying {} to {}.".format(perf_log, results_location))
         shutil.copy2(perf_log, results_location)
-        # archive the ETL file and upload it
-        etl_file = os.path.join(test_location, "latest-perf-build.etl")
-        etl_zip = os.path.join(test_location, "latest-perf-build.zip")
-        log.info("Compressing {} into {}".format(etl_file, etl_zip))
-        zip_script.zipFilesAndFolders(etl_zip, [etl_file], True, True)
-        log.info("Uploading ETL from {}".format(etl_zip))
-        _write_output_path(etl_zip, settings)
+        if xunit_test_type == xunit.XUNIT_CONFIG_PERF_WINDOWS:
+            # only windows runs would generate the etl file
+            # archive the ETL file and upload it
+            etl_file = os.path.join(test_location, "latest-perf-build.etl")
+            etl_zip = os.path.join(test_location, "latest-perf-build.zip")
+            log.info("Compressing {} into {}".format(etl_file, etl_zip))
+            zip_script.zipFilesAndFolders(etl_zip, [etl_file], True, True)
+            log.info("Uploading ETL from {}".format(etl_zip))
+            _write_output_path(etl_zip, settings)
```

Comment:
It might be useful to log what this function returns (so that we know where we put the file), in case we can't find it later for some reason.